Formula/irrlicht.rb

```diff
@@ -73,7 +73,8 @@ def install
 
   test do
     on_macos do
-      assert_match "x86_64", shell_output("lipo -info #{lib}/libIrrlicht.a")
+      cpu = Hardware::CPU.arm? ? "arm64" : "x86_64"
+      assert_match cpu, shell_output("lipo -info #{lib}/libIrrlicht.a")
```

Comment:
```suggestion
      assert_match Hardware::CPU.arch.to_s, shell_output("lipo -info #{lib}/libIrrlicht.a")
```
I _think_ this should be the same thing

Reply:
I'll check, I wasn't sure what to do for this one.

Reply:
LGTM
```
  % git diff
diff --git a/Formula/irrlicht.rb b/Formula/irrlicht.rb
index c3a3070dcf..996fc25ae4 100644
--- a/Formula/irrlicht.rb
+++ b/Formula/irrlicht.rb
@@ -73,8 +73,7 @@ class Irrlicht < Formula
 
   test do
     on_macos do
-      cpu = Hardware::CPU.arm? ? "arm64" : "x86_64"
-      assert_match cpu, shell_output("lipo -info #{lib}/libIrrlicht.a")
+      assert_match Hardware::CPU.arch.to_s, shell_output("lipo -info #{lib}/libIrrlicht.a")
     end
     on_linux do
       cp_r Dir["#{pkgshare}/examples/01.HelloWorld/*"], testpath
  % brew test irrlicht
==> Testing irrlicht
==> lipo -info /opt/homebrew/Cellar/irrlicht/1.8.4/lib/libIrrlicht.a
  % 
```

Reply:
lemme squash that in.

Reply:
squashed.